### PR TITLE
Display human-friendly class and args for ActionMailer jobs

### PR DESCRIPTION
**Pull request by dreyks**

### CODE REVIEW

File: lib/sidekiq/api.rb [hunk 1]
```diff
@@ -283,7 +283,9 @@ def display_class
                      "#{target}.#{method}"
                    end
                  when "ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper"
-                   @item['wrapped'] || args[0]
+                   job_class = @item['wrapped'] || args[0]
+                   # MailerClass#mailer_method
+                   'ActionMailer::DeliveryJob' == job_class ? args[0]['arguments'][0..1].join('#') : job_class
```
Comment (mperham) [inline lib/sidekiq/api.rb]: Could you make this conditional a real if, not a ternary?

File: lib/sidekiq/api.rb [hunk 2]
```diff
@@ -297,7 +299,9 @@ def display_args
                     arg
                   end
                 when "ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper"
-                  @item['wrapped'] ? args[0]["arguments"] : []
+                  job_args = @item['wrapped'] ? args[0]["arguments"] : []
+                  # remove MailerClass, mailer_method and 'deliver_now'
+                  'ActionMailer::DeliveryJob' == (@item['wrapped'] || args[0]) ? job_args.drop(3) : job_args
```
Comment (mperham) [inline lib/sidekiq/api.rb]: likewise

**Comment by dreyks** [comment]
done

**Comment by mperham** [comment]
Thanks!

**Pull request merged**